third_party/2and3/dateutil/rrule.pyi

```diff
@@ -0,0 +1,103 @@
+from ._common import weekday as weekdaybase
+from typing import Any, List, Optional, Union
+import datetime
+
+YEARLY: int
+MONTHLY: int
+WEEKLY: int
+DAILY: int
+HOURLY: int
+MINUTELY: int
+SECONDLY: int
+
+class weekday(weekdaybase):
+    def __init__(self, wkday, n: Optional[Any] = ...) -> None: ...
```

Comment:
You can just omit `__init__` because the argument types are the same as in the base class.